### PR TITLE
Update basic.build

### PR DESCRIPTION
**Pull request by ThebeRising**
Made changes to the build file

### CODE REVIEW

File: basic.build [hunk 1]
```diff
@@ -1,19 +1,38 @@
...
+<property name="browser" value=""/>
+<property name="spec" value=""/>
+<property name="rows" value=""/>
+
     <target name="build">
         <exec program="CMD.EXE"
             workingdir="${project_root}"
-            commandline="/C gauge specs">
-        
+            commandline="/C gauge ${rows} ${browser} specs${spec}">
```
Comment (williamdewitt) [inline basic.build]: I don't think your params are going to work. Gauge will look for switches like --env.

- add check for if parameters != "" then append command with switch and params

Comment (ThebeRising) [inline basic.build]: the commands I use for running are:
nant build -D:browser="chrome" -D:spec="\Spec.spec" 

It works, unless I am not understanding what you are saying?

Comment (williamdewitt) [inline basic.build]: Ok. I'm going to approve and merge this but I need to chat to you about it

Comment (ThebeRising) [inline basic.build]: Alright.

**Pull request merged**